templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -333,39 +326,64 @@ Parameters:
     Default: ""
     Description: 'Use custom existing DNS name for your Data Center instance. This will take precedence over HostedZone. Please note: you must own the domain and configure it to point at the load balancer.'
     Type: String
+  # This parameter will be enabled when support for Aurora is added. Currently "dark" feature.
+  # DBEngine:
+  #   Default: 'PostgreSQL'
+  #   Description: 'Database Engine to use for the application. PostgreSQL or Amazon Aurora PostgreSQL'
+  #   AllowedValues:
+  #     - 'PostgreSQL'
+  #     - 'Amazon Aurora PostgreSQL'
+  #   ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
+  #   Type: String
   DBInstanceClass:
-    Default: db.m4.large
+    Default: db.m5.large
     AllowedValues:
-      - db.m4.large
-      - db.m4.xlarge
-      - db.m4.2xlarge
-      - db.m4.4xlarge
-      - db.m4.10xlarge
-      - db.m4.16xlarge
+      - db.m5.large
+      - db.m5.xlarge
+      - db.m5.2xlarge
+      - db.m5.4xlarge
+      - db.m5.12xlarge
+      - db.m5.24xlarge
+      - r5.large
+      - r5.xlarge
+      - r5.2xlarge
+      - r5.4xlarge
+      - r5.12xlarge
+      - r5.24xlarge
       - db.r4.large
       - db.r4.xlarge
       - db.r4.2xlarge
       - db.r4.4xlarge
       - db.r4.8xlarge
       - db.r4.16xlarge
+      - db.m4.large
+      - db.m4.xlarge
+      - db.m4.2xlarge
+      - db.m4.4xlarge
+      - db.m4.10xlarge
+      - db.m4.16xlarge
+      - db.t3.medium
+      - db.t3.large
+      - db.t3.xlarge
+      - db.t3.2xlarge
       - db.t2.medium
       - db.t2.large
       - db.t2.xlarge
       - db.t2.2xlarge
     ConstraintDescription: Must be a valid RDS instance class, from the selection list
-    Description: RDS instance type
+    Description: RDS instance type # (must be r family if using Aurora)
```

Comment:
Do we need to comment out Aurora here until we are ready to support it?

Reply:
It is isn't it?

Reply:
Misread the comment